Userland/Libraries/LibGfx/SystemTheme.h

```diff
@@ -31,6 +31,24 @@ namespace Gfx {
     C(BaseText)                    \
     C(Button)                      \
     C(ButtonText)                  \
+    C(ColorSchemeBackground)       \
+    C(ColorSchemeForeground)       \
+    C(Black)                       \
+    C(Red)                         \
+    C(Green)                       \
+    C(Yellow)                      \
+    C(Blue)                        \
+    C(Magenta)                     \
+    C(Cyan)                        \
+    C(White)                       \
+    C(BrightBlack)                 \
+    C(BrightRed)                   \
+    C(BrightGreen)                 \
+    C(BrightYellow)                \
+    C(BrightBlue)                  \
+    C(BrightMagenta)               \
+    C(BrightCyan)                  \
+    C(BrightWhite)                 \
```

Comment:
The rest of this macro looks sorted alphabetically, so maybe also sort these?
Or at least put all these at the bottom/top to indicate they are some group